logstash-core/src/main/java/org/logstash/log/LoggerExt.java

```diff
@@ -130,7 +130,7 @@ public IRubyObject rubyTrace(final ThreadContext context, final IRubyObject[] ar
     public static IRubyObject configureLogging(final ThreadContext context, final IRubyObject self,
                                         final IRubyObject args[]) {
         synchronized (CONFIG_LOCK) {
-            RubyString path = args.length > 1 ? (RubyString) args[1] : null;
+            IRubyObject path = args.length > 1 ? args[1] : null;
             String level = args[0].asJavaString();
             try {
                 setLevel(level, (path == null || path.isNil()) ? null : path.asJavaString());
```

Comment:
`path.isNil()` would never be true since only a `RubyNil` can be nil, and the cast earlier breaks if an actual `RubyNil` is passed in.